src/commands/config/list.js

```diff
@@ -0,0 +1,32 @@
+const { BaseCommand } = require('@twilio/cli-core').baseCommands;
+
+const availableConfigs = ['edge'];
+
+class ConfigList extends BaseCommand {
+  async run() {
+    await super.run();
+    const configData = [];
+    availableConfigs.forEach((config) => {
+      configData.push({
+        configName: config,
+        value: this.getFromEnvironment(config) || this.userConfig[config],
+      });
+    });
+    this.output(configData);
+  }
+
+  getFromEnvironment(config) {
+    const configEnv = `TWILIO_${config.toUpperCase()}`;
+    if (process.env[configEnv]) {
+      return `${process.env[configEnv]}[env]`;
+    }
+    return undefined;
+  }
+}
+ConfigList.description = 'list Twilio CLI configurations.';
+
+ConfigList.flags = BaseCommand.flags;
+module.exports = {
+  ConfigList,
+  availableConfigs,
```

Comment:
Should we move `availableConfigs` to a separate configurations file? ConfigList doesn't look the right layer to put these IMO. Thoughts?